README.md

```diff
@@ -47,39 +49,43 @@ The design, organization, and future releases of BlueOS are aligned with these p
 BlueOS is available in three release types:
 
 - **Stable:** Officially tested and validated versions with long-term support. Recommended for most users.
-- **Beta:** Quick-passed rolling releases with new features, bug fixes, and improvements.
-- **Master:** Rapidly-passed bleeding-edge development releases. These are the very latest features that may not have been tested yet.
+- **Beta:** Lightly tested rolling releases with new features, bug fixes, and improvements.
+- **Master:** Bleeding-edge development releases with almost daily changes. These are the very latest features that may not have been tested yet.
 
-## Vehicle Support
+## Vehicle Support 🛸
 
-BlueOS has been designed with a focus on vehicle and to be platform agnosticism. Our aim is to facilitate broad compatibility across a wide spectrum of applications. Currently, BlueOS officially supports the following vehicle types:
+BlueOS has been designed with a focus on vehicle and platform agnosticism. Our aim is to facilitate broad compatibility across a wide spectrum of applications. Currently, BlueOS officially supports the following vehicle types:
 
 ### **Boats (ArduRover)**
 
-ArduRover is an open-source, unmanned boat platform. Whether you are commanding a leisure boat or a research vessel, BlueOS's compatibility with ArduRover ensures that you can navigate the waters smoothly 🌊.
+ArduRover is an open-source, uncrewed boat platform. Whether you are commanding a leisure boat or a research vessel, BlueOS's compatibility with ArduRover ensures that you can navigate the waters smoothly 🌊.
+
+[BlueBoat](https://bluerobotics.com/store/blueboat/blueboat/) is supported by default.
 
 ![](doc/blueboat.png)
 
 ### **Submarines (ArduSub)**
 
-ArduSub is the go-to control system for remotely operated underwater vehicles (ROVs) 🐟. BlueOS offers seamless integration with ArduSub, enabling efficient management and operation of underwater vehicles. Right now it provides support for BlueROV2 out of the box.
+ArduSub is the go-to control system for remotely operated underwater vehicles (ROVs) 🐟. BlueOS offers seamless integration with ArduSub, enabling efficient management and operation of underwater vehicles. 
+
+[BlueROV2](https://bluerobotics.com/store/rov/bluerov2/) is supported out of the box.
 
 ![](doc/bluerov.png)
 
 ### **Generic (ArduPilot / PX4)**
 
-For a wide array of unmanned vehicles, whether terrestrial, aerial, or marine, BlueOS provides support for the generic ArduPilot and PX4 autopilots. This extends the range of vehicles that can be managed using our system, from drones to autonomous cars and more.
+BlueOS provides generic support for a wide variety of terrestrial, aerial, and marine unmanned vehicles that use the  ArduPilot and PX4 autopilots. This extends the range of vehicles that can be managed using our system, from drones to autonomous cars and more.
```

Comment:
as you've changed above, maybe do it here as well

```suggestion
BlueOS provides generic support for a wide variety of terrestrial, aerial, and marine uncrewed vehicles that use the  ArduPilot and PX4 autopilots. This extends the range of vehicles that can be managed using our system, from drones to autonomous cars and more.
```

Reply:
Lol, I agonised over that for a bit but decided not to because it sounded too weird with the original phrasing. With the current phrasing I think that's ok, so I've changed it 👍🏻